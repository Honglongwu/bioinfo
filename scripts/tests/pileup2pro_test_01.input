seq1	15316	C	9	,,,,.,.,.	eicbidf`f
seq1	15317	C	10	,,,,.,.,.^],	eieTi_fefd
seq1	15318	A	10	,,,,.,.,.,	eicaigfefc
seq1	15319	A	10	,,,,.,.,.,	eia\hbfefb
seq1	15320	A	10	,,,,.,.,.,	eiebi\fdfb
seq1	15321	A	10	,,,,.,.,.,	biedi\fef`
seq1	15322	T	10	,,,,.,.,.,	biebhVfcfb
seq1	15323	T	11	,$,,,.,.,.,^].	bieYiXfffde
seq1	15324	T	10	,,,.,.,.,.	gechb\ff\f
seq1	15325	C	10	,,,.,.,.,.	gfbihfffcf
seq1	15326	C	10	,,,.,.,.,.	gecihfffdf
seq1	15327	T	10	,,,.,.,.,.	gfaggfffff
seq1	15328	G	10	,,,.,.,.,.	gd_hcfffff
seq1	15329	A	10	,,,.,.,.,.	cf_iecefdf
seq1	15330	G	10	,,,.,.,.,.	cdVhafefaf
seq1	15331	T	10	,,,.,.,.,.	^f\Y\dfddf
seq1	15332	A	10	,,,.,.,.,.	cfUa\fffef
seq1	15333	T	10	,,,.,.,.,.	cdec_feecf
seq1	15334	T	10	,,,.,.,.,.	[fdhcfffef
seq1	15335	T	10	,,,.,.,.,.	[efi^effcf
seq1	15336	T	10	,$,,.,.,.,.	_ff`Xfffaf
seq1	15337	G	9	,,.,.,.,.	fceXfefbf
seq1	15338	C	9	,,.,.,.,.	cfgccefff
seq1	15339	A	9	,,.,.,.,.	ffbgfffdf
seq2	1	A	9	gggg.,.,.	fcgffffff
seq2	2	G	9	,,.,.aAA.	ffccffeff
seq2	3	A	9	,,.,.,.,.	ffgieefff
seq2	4	C	10	,,.,.,.,.^].	fehhfffeff
seq2	5	A	10	,,.,.,.,..	feigcfeaff
seq2	6	T	10	,,.,.,.,..	dcieefffff
seq2	7	G	10	,,.,.,.,..	feh^efffff
seq2	8	A	10	,,.,.,.,..	fdhcfffeff
seq2	9	G	11	,,TT.,.,TT^]T	f^ibfffffd_
seq2	10	C	12	,,.,.,.,...^].	faif`fffffbg
seq2	11	T	12	,,.,.,.,....	`fhhfffecfbg
seq2	12	T	12	,,.,.,.,....	cfigfcffffed
seq2	13	G	12	,,.,.,.,....	ffifffffffeg
seq2	14	C	12	,,.,.,.,....	feifffffffeg
seq2	15	T	12	,,.,.,.,....	fcbgfeffffcg
seq2	16	G	12	,,.,.,.,....	ffgdeffffeeg
seq3	1	G	12	,,.,.,.,....	ffhbffffeffg
seq3	2	T	12	,,$ccccc,....	ffbcddcefcge
seq3	3	G	11	,.,.,.,....	edbeffffegg
seq3	4	A	11	,.,.,.,....	ff^afeffffg
seq3	5	A	11	,.,.,.,....	fgP`dfdcfgg
seq3	6	C	11	,.,.,.,....	ffRafcfffig
seq3	7	C	11	,.,.,.,....	fgZefffffig
seq3	8	T	11	,.,.,.,....	fd_efffffig
seq3	9	T	11	,.,.,.,....	fd_effffeig
seq3	10	G	11	,.,.,.,....	fddcffffddg
