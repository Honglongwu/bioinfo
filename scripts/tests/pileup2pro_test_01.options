pileup2pro_test_01.input
