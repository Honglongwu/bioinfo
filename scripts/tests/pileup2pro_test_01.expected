>seq1
15316	0	9	0	0
15317	0	10	0	0
15318	10	0	0	0
15319	10	0	0	0
15320	10	0	0	0
15321	10	0	0	0
15322	0	0	0	10
15323	0	0	0	11
15324	0	0	0	10
15325	0	10	0	0
15326	0	10	0	0
15327	0	0	0	10
15328	0	0	10	0
15329	10	0	0	0
15330	0	0	10	0
15331	0	0	0	10
15332	10	0	0	0
15333	0	0	0	10
15334	0	0	0	10
15335	0	0	0	10
15336	0	0	0	10
15337	0	0	9	0
15338	0	9	0	0
15339	9	0	0	0
>seq2
1	5	0	4	0
2	3	0	6	0
3	9	0	0	0
4	0	10	0	0
5	10	0	0	0
6	0	0	0	10
7	0	0	10	0
8	10	0	0	0
9	0	0	6	5
10	0	12	0	0
11	0	0	0	12
12	0	0	0	12
13	0	0	12	0
14	0	12	0	0
15	0	0	0	12
16	0	0	12	0
>seq3
1	0	0	12	0
2	0	5	0	7
3	0	0	11	0
4	11	0	0	0
5	11	0	0	0
6	0	11	0	0
7	0	11	0	0
8	0	0	0	11
9	0	0	0	11
10	0	0	11	0
